plugins/events-backend/src/service/http/HttpPostIngressEventPublisher.ts

```diff
@@ -108,32 +102,17 @@ export class HttpPostIngressEventPublisher {
     const logger = this.logger;
 
     router.post(path, async (request, response) => {
-      const requestBody = request.body;
-      if (!Buffer.isBuffer(requestBody)) {
-        throw new Error(
-          `Failed to retrieve raw body from incoming event for topic ${topic}; not a buffer: ${typeof requestBody}`,
-        );
-      }
-
-      const bodyBuffer: Buffer = requestBody;
-      const parsedContentType = contentType.parse(request);
-      if (
-        !parsedContentType.type ||
-        parsedContentType.type !== 'application/json'
-      ) {
-        throw new UnsupportedMediaTypeError(parsedContentType.type);
-      }
+      const requestContentType = contentType.parse(request);
+      const bodyParser = this.bodyParsers[requestContentType.type];
```

Comment:
As of the previous code, `parsedContentType.type` can be undefined which we should handle here, too.

Reply:
I didn't find a way the type could be undefined [here](https://github.com/jshttp/content-type/blob/master/index.js#L106), but in any case I put a fallback just to be sure